internal/services/springcloud/migration/gateway_route_config_migration_v0_to_v1.go

```diff
@@ -38,9 +39,11 @@ func (s SpringCloudGatewayRouteConfigV0ToV1) Schema() map[string]*pluginsdk.Sche
 			},
 		},
 
+		// lintignore:S013
 		"protocol": {
 			Type:     pluginsdk.TypeString,
-			Required: false,
+			Optional: !features.FourPointOh(),
+			Required: features.FourPointOh(),
```

Comment:
this is a state migration which is meant to be a point in time reference, so this'd want to be:
```suggestion
			Optional: true,
```